disnake/member.py

```diff
@@ -667,6 +667,18 @@ def top_role(self) -> Role:
 
         return max(guild.get_role(rid) or guild.default_role for rid in self._roles)
 
+    @property
+    def display_icon(self) -> Optional[Asset]:
+        """Optional[:class:`Asset`]: Returns the member's displayed role icon, if any.
+        .. versionadded:: 2.5
```

Comment:
```suggestion

        .. versionadded:: 2.5
```

Reply:
discord seems to be adding some sort of system icons that display on members. Naming this icon may interfere with that.

Perhaps this could be named role_icon?